### ISSUE TITLE
`vue create` does only create package.json & package-lock.json

### ISSUE THREAD

**Issue by baermathias** [vuejs/vue-cli#3669]
### Version
3.5.1

### Environment info
```
  System:
    OS: Linux 5.0 Ubuntu 18.04.2 LTS (Bionic Beaver)
    CPU: (8) x64 Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz
  Binaries:
    Node: 10.15.0 - ~/.nvm/versions/node/v10.15.0/bin/node
    Yarn: Not Found
    npm: 6.4.1 - ~/.nvm/versions/node/v10.15.0/bin/npm
  Browsers:
    Chrome: 72.0.3626.109
    Firefox: 65.0.1
  npmGlobalPackages:
    @vue/cli: 3.5.1


```

### Steps to reproduce
Install vue: 
`npm install -g @vue/cli`

create project: 
`vue create projectname`

go to project dir and run: 
`cd projectname` 

`npm run serve`

### What is expected?
A running project

### What is actually happening?
Getting error message:
npm ERR! missing script: serve

vue has only created 3 files after `vue create projectname`: package.json, package-lock.json, README.md

If I try to create the project with `vue ui`, the same files are being created but a different error message in the browser:
`NO_MODULES`

**Comment by LinusBorg** [comment]
Possibly related to #3664

**Comment by baermathias** [comment]
Setting NODE_ENV to development solves the issue:
`export NODE_ENV=development`

But then another error appears: 
```
ERROR  Failed to compile with 1 errors 
error  in ./src/main.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: [BABEL] ./src/main.js: Cannot find module '@babel/preset-env/data/built-ins.json'
```

**Comment by baermathias** [comment]
Solved it by changing the following setting in `babel.config.js`:

```
module.exports = {
  presets: [
    '@vue/app'
  ]
}
```

to:

```
module.exports = {
  presets: [
    '@babel/preset-env'
  ]
}
```

**Comment by sodatea** [comment]
FYI it's due to the release of babel v7.4.0 and is fixed in v7.4.1
See https://github.com/vuejs/vue-cli/issues/3671
